pkg/controller/flink/client/api.go

```diff
@@ -66,49 +67,53 @@ type FlinkJobManagerClient struct {
-		savepointJobFailureCounter:   labeled.NewCounter("savepoint_job_failed", "Savepoint job request failed", flinkJmClientScope),
+		scope:                              scope,
+		submitJobSuccessCounter:            labeled.NewCounter("submit_job_success", "Flink job submission successful", flinkJmClientScope),
+		submitJobFailureCounter:            labeled.NewCounter("submit_job_failure", "Flink job submission failed", flinkJmClientScope),
+		cancelJobSuccessCounter:            labeled.NewCounter("cancel_job_success", "Flink job cancellation successful", flinkJmClientScope),
+		cancelJobFailureCounter:            labeled.NewCounter("cancel_job_failure", "Flink job cancellation failed", flinkJmClientScope),
+		forceCancelJobSuccessCounter:       labeled.NewCounter("force_cancel_job_success", "Flink forced job cancellation successful", flinkJmClientScope),
+		forceCancelJobFailureCounter:       labeled.NewCounter("force_cancel_job_failure", "Flink forced job cancellation failed", flinkJmClientScope),
+		checkSavepointSuccessCounter:       labeled.NewCounter("check_savepoint_status_success", "Flink check savepoint status successful", flinkJmClientScope),
+		checkSavepointFailureCounter:       labeled.NewCounter("check_savepoint_status_failure", "Flink check savepoint status failed", flinkJmClientScope),
+		getJobsSuccessCounter:              labeled.NewCounter("get_jobs_success", "Get flink jobs succeeded", flinkJmClientScope),
+		getJobsFailureCounter:              labeled.NewCounter("get_jobs_failure", "Get flink jobs failed", flinkJmClientScope),
+		getJobConfigSuccessCounter:         labeled.NewCounter("get_job_config_success", "Get flink job config succeeded", flinkJmClientScope),
+		getJobConfigFailureCounter:         labeled.NewCounter("get_job_config_failure", "Get flink job config failed", flinkJmClientScope),
+		getClusterSuccessCounter:           labeled.NewCounter("get_cluster_success", "Get cluster overview succeeded", flinkJmClientScope),
+		getClusterFailureCounter:           labeled.NewCounter("get_cluster_failure", "Get cluster overview failed", flinkJmClientScope),
+		getCheckpointsSuccessCounter:       labeled.NewCounter("get_checkpoints_success", "Get checkpoint request succeeded", flinkJmClientScope),
+		getCheckpointsFailureCounter:       labeled.NewCounter("get_checkpoints_failed", "Get checkpoint request failed", flinkJmClientScope),
+		savepointJobSuccessCounter:         labeled.NewCounter("savepoint_job_success", "Savepoint job request succeeded", flinkJmClientScope),
+		savepointJobFailureCounter:         labeled.NewCounter("savepoint_job_failed", "Savepoint job request failed", flinkJmClientScope),
+		getCheckpointsConfigSuccessCounter: labeled.NewCounter("get_checkpoints_config_success", "Get checkpoint config request succeeded", flinkJmClientScope),
+		getCheckpointsConfigFailureCounter: labeled.NewCounter("get_checkpoints_config_failed", "Get checkpoint config request failed", flinkJmClientScope),
 	}
```

Comment:
I love `go fmt`, but it can also be disruptive causing PR's to bloat changes and hides the actual diff. 

I could add some line breaks to separate out the new, longer struct member and minimize the line change count on things like this.